A3-Antistasi/resourcecheck.sqf

```diff
@@ -157,7 +157,20 @@ while {true} do
 			};
 		};
 	sleep 3;
-	if ((count antennasDead > 0) and (not(["REP"] call BIS_fnc_taskExists))) then
+	_wreckedAntennas = false;
+	_wc = count antennasDead;
+	if (_wc > 0) then {_wreckedAntennas = true};
+	_smallWar = false;
+	_bigWar = false;
+	if ((tierWar<=3)and{not(["REP"] call BIS_fnc_taskExists)}) then
+	{_smallWar = true};
+	if ((tierWar>3)and{(count(["REP"] call BIS_fnc_taskExists)>1)}) then
+	{_bigWar = true};
+	_xchance = (tierWar*20);
+	_xroll = round(random 100);
+	_final = false;
+	if (_xroll<_xchance) then {_final = true};
+	if ((_wreckedAntennas)and((_smallWar)or(_bigWar))and(_final)) then
```

Comment:
Does it actually make a difference if it's a small or a big way? Isn't this essentially a check for 'There isn't exactly 1 repair mission currently, and there's broken antennas, and our dice roll chance has occurred?'

Reply:
the goal here is that if War Level is 3 or less (smallWar=True) then there can only be 1 repair mission running. If War Level is 4 or 5 then there can be two concurrent repair missions running, since OCU would value them higher during an escalating conflict.

Reply:
Gotcha, makes sense now - I misread it. Though I don't think there can ever be more than 1 Repair mission running? Because any new repair mission spawned will replace the old one in the task list.